tests/test_helpers.py

```diff
@@ -47,7 +47,7 @@
     (["helper", "l_contacts", "run", "-n", "ny", "-i", "col", "-o", "coffee", "-d", "2019-01-15", "-r", "2"],
      ""
      ),
-    (["helper", "u_logurl", "20ly_newprojectum", "this is a test pt.2"],
```

Comment:
I think we can have a test for the list *and* for the update. We only can't have two tests that update the database.